src/it/streamingcommunity/src/eu/kanade/tachiyomi/animeextension/it/streamingcommunity/StreamingCommunity.kt

```diff
@@ -252,68 +256,89 @@ class StreamingCommunity : ConfigurableAnimeSource, AnimeHttpSource() {
 
     // ============================ Video Links =============================
 
-    override fun videoListRequest(episode: SEpisode): Request = GET("$baseUrl/watch/${episode.url}", headers)
+    object VideoRegex {
+        val PLAYLIST_URL_REGEX = Regex("""url: ?'(.*?)'""")
+        val EXPIRES_REGEX = Regex("""'expires': ?'(\d+)'""")
+        val TOKEN_REGEX = Regex("""'token': ?'([\w-]+)'""")
+        val TOKEN_QUALITY_REGEX = Regex("""'token(\d+p?)': ?'([\w-]+)'""")
+        val SUBTITLES_REGEX = Regex("""#EXT-X-MEDIA:TYPE=SUBTITLES.*?NAME="(.*?)".*?URI="(.*?)"""")
```

Comment:
Move these to the companion object and make them private

Reply:
ok